pkg/goclient/goclient.go

```diff
@@ -0,0 +1,177 @@
+package goclient
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/intel/afxdp-plugins-for-kubernetes/constants"
+	"github.com/intel/afxdp-plugins-for-kubernetes/internal/host"
+	"github.com/intel/afxdp-plugins-for-kubernetes/internal/uds"
+)
+
+var (
+	hWR       uds.Handler
+	hPod      host.Handler
+	cleanupG  uds.CleanupFunc
+	connected bool = false
+)
```

Comment:
Just have think about variable names.

Not sure what hWR refers to. but if your to call it uds, then uds.read and uds.write makes an awful lot more sense in the code below.

 Not sure what the G in cleanupG refers to.
 
 hPod I guess kind of makes sense, I'm guessing "host pod", but then don't be afraid to call it hostPod.
 
 Spell it out for people :)